src/main/scala/EmrSparkPlugin.scala

```diff
@@ -57,6 +63,8 @@ object EmrSparkPlugin extends AutoPlugin {
     val sparkTerminateCluster = taskKey[Unit]("terminate cluster")
     val sparkSubmitJob = inputKey[Unit]("submit the job")
     val sparkSubmitJobWithMain = inputKey[Unit]("submit the job with specified main class")
+    val sparkSubmitSteps = taskKey[Unit]("submit multiple steps")
+    val sparkUploadJarToS3 = taskKey[Unit]("Upload jar to S3")
```

Comment:
Are you going to upload multiple **different** jars and execute them in one shot by the `sparkSubmitSteps`?
Or would you like to upload the jar without triggering EMR?
If not, maybe we can just upload the jar in `def submitSteps()` to avoid someone who forget to upload the jar before submitting steps?